src/app/components/text-input-field.tsx

```diff
@@ -58,11 +57,6 @@ export function TextInputField({
         }}
         border="default"
         borderRadius="sm"
-        className={css({
```

Comment:
I just got rid of this as it wasn't working and hasn't been for a while. I opened a task for it https://github.com/leather-wallet/extension/issues/4583